README.md

```diff
@@ -215,7 +215,7 @@ app.use exprestive
   * default: `__dirname`
 * `controllersPattern`
   * [Glob](https://github.com/isaacs/node-glob) pattern used to find controllers. Resolved relative to `appDir`
-  * default: `'controllers/*_controller.{coffee,js}'`
+  * default: `'controllers/*controller.+([^.])'`
```

Comment:
I don't understand this regex. I would expect:

`'controllers/*controller\.([^.]*)'`